colin-api/src/colin_api/models/business.py

```diff
@@ -568,6 +568,23 @@ def get_resolutions(cls, cursor, corp_num: str) -> List:
             current_app.logger.error(f'Error looking up resolution dates for {corp_num}')
             raise err
 
+    @classmethod
+    def get_court_order_date(cls, cursor, event_id: str) -> str:
+        """Get order date for a filing."""
+        try:
+            cursor.execute(
+                """
+                select TO_CHAR(resolution_dt, 'YYYY-mm-dd') from resolution
+                where start_event_id = :event_id
+                """,
+                event_id=event_id
+            )
+            court_order_date = cursor.fetchone()[0]
```

Comment:
Will this throw an error if there is no court order associated with the event? I can't remember if cursor.fetchone() returns null or [] if it doesn't find anything. 
This will be called for every filing with court order in the schema so it'll need to handle this without error

Reply:
Yes, you're right. I'll take a look at this! Thanks!

Reply:
I checked and, yes, it raises a 'NoneType' object is not subscriptable. I fix it. Thanks!